documentation/api/query/v4/index.markdown

```diff
@@ -0,0 +1,66 @@
+---
+title: "PuppetDB 3.2 » API » v4 » Root Endpoint"
+layout: default
+canonical: "/puppetdb/latest/api/query/v4/index.html"
+---
+
+[curl]: ../curl.html#using-curl-from-localhost-non-sslhttp
+[paging]: ./paging.html
+[query]: ./query.html
+[from]: ./operators.html#context-operators
+[entities]: ./entities.html
+
+*Note:* This endpoint is experimental. It may be altered or removed in a future release.
+
+The root query endpoint can be used to retrieve any known entities, from a
+single endpoint.
+
+## `/pdb/query/v4`
+
+This will return any known entity based on the required `query` field. Unlike
+other endpoints, the [entity][entities] must be supplied using a query with the [`from`][from]
+operator.
+
+### URL Parameters
```

Comment:
We should also have links here to the entities and their available query fields.